dist/main.js

```diff
@@ -0,0 +1,40 @@
+const renderer = new Renderer();
+const model = new Model();
+
+
+const loadPage = async () => {
+    await model.getDataFromDB()
+    await model.getCurrentCity()
+    renderer.render(model.cityData)
+}
+loadPage()
+
+const handleSearch = async cityInput => {
+    await model.getCityData(cityInput, "", "")
```

Comment:
it's a convention that if you don't want to send some parameters to a function, send "_" (instead of ""). it's not very important but still, that's the convention.
Besides that, in this case, because the parameters of lat and lon come after the first one - you don't need to send them at all (just invoke it with the first one) - then they would be undefined in the getCityData method